app/src/main/java/com/jnj/vaccinetracker/common/validators/PhoneValidator.kt

```diff
@@ -10,7 +8,8 @@ class PhoneValidator @Inject constructor(private val phoneNumberUtil: PhoneNumbe
         /**
          * only numbers and white spaces
          */
-        private val phoneRegex = "[0-9\\s]*".toRegex()
+        private val phoneRegex = "^(07|02|03)[0-9\\s]*$".toRegex()
```

Comment:
what phone number does this regex allow?

Reply:
This allows, 07, 02 and 03 numbers, as long as user enters 07++++, it should be valid.